CMakeLists.txt

```diff
@@ -166,6 +167,10 @@ if(FLATBUFFERS_CODE_COVERAGE)
       "${CMAKE_EXE_LINKER_FLAGS} -fprofile-arcs -ftest-coverage")
 endif()
 
+if(FLATBUFFERS_DEBUG)
```

Comment:
debug mode is the default in CMake. Why is this added? Seems unrelated to `--size-prefixed` so should be in a different PR if at all.

Reply:
Ok, I've removed this, as it is completely useless :-)